app/jsx/components/TabAuthorComp.jsx

```diff
@@ -44,7 +44,8 @@ class TabAuthorComp extends React.Component {
       {!p.attrs['orig_citation'] &&
         <div className="c-itemactions">
           <div className="o-download">
-            <NotYetLink element="button" className="o-download__button">Download Citation</NotYetLink>
+            {/* ToDo: Link up citation */}
+            <a href="" className="o-download__button" download="">Download Citation</a>
```

Comment:
This should probably be a <NotYetLink> right?

Reply:
@martinhaye  Not clear on your question.

Reply:
Oops, github edited out my tag. I think it should be a NotYetLink, right?